lms/static/sass/views/_homepage.scss

```diff
@@ -32,6 +32,7 @@ $learn-more-horizontal-position: calc(50% - 100px); // calculate the left positi
 
       .course-image .cover-image {
         height: $course-image-height;
+        background: $uxpl-blue-base;
```

Comment:
No, because the button is white. It needs to be something that will provide enough contrast for white text.